src/components/v2v3/V2V3Project/V2V3ProjectSettings/hooks/useReconfigureFundingCycle.ts

```diff
@@ -110,8 +114,11 @@ export const useReconfigureFundingCycle = ({
         : editingFundingCycleMetadata
 
       const weight = getWeightArgument({
-        currentFundingCycleWeight: fundingCycle.weight,
-        newFundingCycleWeight: editingFundingCycleData.weight,
+        currentWeightAfterDiscountRate: deriveNextIssuanceRate({
```

Comment:
This function is an incredible bong but used in a bunch of other places, so I don't think it's worth reworking.

Reply:
But basically, if given these arguments, it gives back that derived mint rate reliably.